app/models/widget-discussions.js

```diff
@@ -52,7 +52,11 @@ export default EmberObject.extend(
 						name: createdBy.name,
 						profileUrl: buildUrl({
 							namespace: 'User',
-							title: createdBy.name
+							title: createdBy.name,
+							relative: true,
+							query: {
+								useskin: 'oasis'
```

Comment:
Why do we need it on mobile?

Reply:
to avoid some round trips 
with this we'll go straight to oasis